src/client/components/TotalPriceCard/TotalPriceCard.styles.css

```diff
@@ -1,10 +1,9 @@
-.wrapper {
+.wrapperTotalPrice {
```

Comment:
```suggestion
.wrapper-total-price {
```

Use this other syntax instead. Apply the same to the rest of the names

Reply:
I have used kebab-case as per your suggestion on all the css elements 👍